### PR TITLE
Crafting Station and Inventory

### PR DESCRIPTION
**Pull request by LPGMankato**
image

### CODE REVIEW

File: Assets/Scripts/PlayerControlsBuilder.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public override List<KeyCode> SetPlayerControls()
         listOfKeys.Add(right);
         listOfKeys.Add(jump);
 
-        Debug.Log("Set Contols for arrows.");
```
Comment (ellie-jean) [inline Assets/Scripts/PlayerControlsBuilder.cs:34]: This debug can be removed

File: Assets/Scripts/PlayerControlsBuilder.cs [hunk 2]
```diff
@@ -59,7 +59,7 @@ public override List<KeyCode> SetPlayerControls()
         listOfKeys.Add(right);
         listOfKeys.Add(jump);
 
-        Debug.Log("Set Contols for wasd.");
```
Comment (ellie-jean) [inline Assets/Scripts/PlayerControlsBuilder.cs:62]: This debug can be removed

File: Assets/Scripts/OptionMenuSetControls.cs [hunk 3]
```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections;
```
Comment (ellie-jean) [inline Assets/Scripts/OptionMenuSetControls.cs]: This class can be delete

File: Assets/Scripts/OptionMenuSetControls.cs.meta [hunk 4]
```diff
@@ -0,0 +1,11 @@
+fileFormatVersion: 2
```
Comment (ellie-jean) [inline Assets/Scripts/OptionMenuSetControls.cs.meta]: This class meta can be removed

**Review by ellie-jean — APPROVED** [review]

**Review by ellie-jean — APPROVED** [review]
Looks good!

**Comment by LPGMankato** [comment]
Thanks Ellie! The only issue I could see in the future is with the inventory canvas and how close the items are. For instance, the names of the items might overlap with the current spacing so we might have to widen the spacing between the items.

**Pull request merged**